uiowa/uiowa
# [INRC] Grants

Resolves #7821





# How to test


```
ddev blt ds --site=inrc.law.uiowa.edu && ddev drush @lawinrc.local config:import --source ../config/sites/inrc.law.uiowa.edu --partial -y && ddev drush @lawinrc.local cim -y && ddev drush @lawinrc.local uli
```
Check the Counties and Cities taxonomies--they should be populated with values.
Add a few terms to each of the other site-specific vocabs.
Create a Grant
Compare the fields/options with what is detailed in #7821 
Check against our content type best practices and guidelines: https://docs.prod.drupal.uiowa.edu/development-workflow/content-type-best-practices
